sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java

```diff
@@ -280,6 +277,20 @@ private void initDataReader(
     }
   }
 
+    private ValuesReader getValuesReader(Encoding encoding) {
+        switch (encoding) {
```

Comment:
Could you use `two-space` indentation like the other part of this file, @parthchandra ?